script.js

```diff
@@ -0,0 +1,53 @@
+// var choices = [1, 2, 3, 4, 5, 6];
+//maybe change "choices" to "Prompts" and the oucomes of prompts are the parameteres for moving on
+
+// var bluePill = false;
+// var redPill = true;
+
+
+
+var ready = prompt("Wake up, Neo. Are you ready?");
+
+ if (ready != null) {
+    confirm("Great. Follow the white rabbit");
+
+  }
+  else {
```

Comment:
make sure to indent correctly to keep this in the same tab length as the if condition it's associated with.